src/BenchmarksDriver/Program.cs

```diff
@@ -413,6 +438,7 @@ public static int Main(string[] args)
                         [Id] [int] IDENTITY(1,1) NOT NULL PRIMARY KEY,
                         [DateTime] [datetimeoffset](7) NOT NULL,
                         [Scenario] [nvarchar](max) NOT NULL,
+                        [TargetFramework] [nvarchar](max) NOT NULL,
```

Comment:
Yes.  Right before this changed is merged to dev, I will do the following:

1. Add `Frameworks` column to existing table with `NOT NULL` and a default value of `Core`.
2. Existing rows are assigned default value.
3. Remove default value, so future inserts will be required to specify `Core` or `Desktop`.